### PR TITLE
Adds MSRV ci check

### PR DESCRIPTION
**Pull request by Superhepper**
Signed-off-by: Jesper Brynolf <jesper.brynolf@gmail.com>

### CODE REVIEW

**Review by ionut-arm — APPROVED** [review]
Thanks for the patch!! Could you also add a section in the README of the `tss-crate` about the MSRV?

**Comment by Superhepper** [comment]
Yes, I will update the readme as well. I have not tested this properly so I wonder if this works.

**Comment by ionut-arm** [comment]
If you merge or push it to your fork's `main` branch, you can run the whole `nightly` suite manually from [here](https://github.com/Superhepper/rust-tss-esapi/actions/workflows/nightly.yml) (though for now it seems disabled)

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -51,3 +51,15 @@ jobs:
         run: docker run -v $(pwd):/tmp/rust-tss-esapi -w /tmp/rust-tss-esapi/tss-esapi --security-opt seccomp=unconfined ubuntucontainer /tmp/rust-tss-esapi/tss-esapi/tests/coverage.sh
       - name: Collect coverage results
         run: bash <(curl -s https://codecov.io/bash)
+
+  msrv:
+    name: Check minimum supported rust version (MRVS)
```
Comment (wiktor-k) [inline .github/workflows/nightly.yml:56]: Typo: `MSRV`

**Review by wiktor-k — APPROVED** [review]
Looks good :+1:

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (Superhepper) [reply .github/workflows/nightly.yml:56]: Fixed

**Comment by Superhepper** [comment]
This won't work. MSRV needs to be checked in the container or the build will fail. I will put this as a draft while I fix that.

**Pull request merged**